napari/utils/theme.py

```diff
@@ -135,7 +134,7 @@ def increase(font_size: str, pt: int) -> str:
     return f'{int(font_size[:-2]) + int(pt)}pt'
 
 
-def _parse_color_as_rgb(color: str | Color) -> tuple[int, int, int]:
+def _parse_color_as_rgb(color: str | Color) -> tuple | Any:
```

Comment:
Can you (or anyone else) explain this change?
Seems like the actual function will always return a 3-tuple of ints?
But I'm far from a typing expert, so would like better understanding.

Reply:
Good catch @psobolewskiPhD. I think I relaxed it when debugging and forgot to reset it. Let's see if the new commit fixes this and another in the same file as well.